tests/flow/test_timeout.py

```diff
@@ -10,7 +11,7 @@
 class testQueryTimeout(FlowTestsBase):
     def __init__(self):
         # skip test if we're running under Valgrind
-        if Env().envRunner.debugger is not None:
+        if Env().envRunner.debugger is not None or os.getenv('COV') == '1':
```

Comment:
there are other tests which perform skip if they run under debugger,
should we apply the same change to them as well ?

Reply:
Looked them over, they are doing so because they fail to pass logically because of valgrind or that they are causing leaks. I think it's the only spot where time comes into play.